moveit_experimental/moveit_jog_arm/src/cpp_interface_example/cpp_interface_example.cpp

```diff
@@ -109,8 +110,14 @@ int main(int argc, char** argv)
 
   // Retrieve the current joint state from the jogger
   sensor_msgs::JointState current_joint_state = jog_interface.getJointState();
+  ROS_INFO_STREAM("Current joint state:");
```

Comment:
Always use `_NAMED` so that the user can selectively filter out msgs with the rosconsole

Reply:
done